modules/instruments/test/TestPlan.md

```diff
@@ -43,3 +43,13 @@ instrument for that candidate. Enter a date (same a step 4) posterior to the DoD
 12. Verify Validity flags are functioning for instruments.
     - **Validity:** can be marked as “Valid”, “Questionable”, or “Invalid”. Whether or not this flag is shown for an instrument is by the boolean $ValidityEnabled. Whether the field is required before flagging an instrument as complete is determined by $ValidityRequired.
     - You can test the forgoing flags by switching the corresponding boolean of the test instruments to either `true` or `false`.
+13. Dashboard Widget - "My Tasks" for Incomplete forms
```

Comment:
Can you put these in different sections? It's not clear the way it's written that the user needs to go to a different module to validate these things.

Reply:
@driusan done